.github/workflows/codeql.yml

```diff
@@ -0,0 +1,60 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: 'CodeQL'
+
+on:
+  push:
+    branches: ['development', feature/*, hotfix/*, main, release/*]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: ['development', feature/*, hotfix/*, release/*]
+  schedule:
+    - cron: '30 3 * * 4'
```

Comment:
Was there a reason why a schedule for this action to run on Thursday a 03:30 was used 😅 .

Reply:
there are many jobs, and I am configuring each to not have conflict.